lib/scope.js

```diff
@@ -277,47 +277,47 @@ function defineScope(cls, targetClass, name, params, methods, options) {
 
   cls['__get__' + name] = fn;
 
-  var fn_create = function () {
+  var fnCreate = function() {
     var f = this[name].create;
     f.apply(this[name], arguments);
   };
```

Comment:
I think this is to prevent collision with `DataAccessObject.create = function create(...) { ... }`.